test-cases/longevity/longevity-add-remove-dc.yaml

```diff
@@ -0,0 +1,18 @@
+test_duration: 60
+stress_cmd: ["cassandra-stress write cl=QUORUM duration=30m -schema 'replication(factor=3) compaction(strategy=SizeTieredCompactionStrategy)' -port jmx=6868 -mode cql3 native -rate threads=5 -pop seq=1..10000000 -log interval=5",
+             "cassandra-stress counter_write cl=QUORUM duration=30m -schema 'replication(factor=3) compaction(strategy=DateTieredCompactionStrategy)' -port jmx=6868 -mode cql3 native -rate threads=5 -pop seq=1..10000000"
+             ]
+
+n_db_nodes: 3
+n_loaders: 1
+instance_type_db: 'i3.large'
+instance_type_loader: 'c5.large'
+n_monitor_nodes: 0
```

Comment:
why 0 monitors?

Reply:
mistake. when developing I didn't use it - but you reminded me that I need to update monitoring stack to show added DC.